### ISSUE TITLE
ValueError on entering edit mode

### ISSUE THREAD

**Issue by nschneid** [nschneid/activedop#95]
Tokenized sentence: "funds obligated for any program year may be expended by the State during that program year and the two succeeding program years and no amount shall be deobligated on account of a rate of expenditure which _. is consistent with the program plan ."

**Comment by nschneid** [comment]
I tried to debug this a bit. The error comes from within CGEL tree validation of the parser-generated tree. It seems there is a nonterminal in the CGEL tree that is the `head` of another node, yet looking up its children produces an empty list.

I printed out the CGEL-format tree, saved it to a file and ran validation on it—which did not encounter this error. I wonder if it is a bug in the code that converts the parser output to a CGEL tree.
